frontend/src/components/dashboard/CodeComparison.tsx

```diff
@@ -403,16 +433,43 @@ const CodeComparison = ({
           solutionId={modalDataPoint?.analysis.solutionId}
           footer={
             <ModalFooter>
```

Comment:
Visually, I'm wondering if this should go into the footer or the header.
It seems like the header would be a better fit 🤔

Reply:
HEP agrees. I will be interested on Lucid thoughts :)